web/styles/settings.css

```diff
@@ -358,6 +358,40 @@ td .button {
     }
 }
 
+.loading-text {
+    position: absolute;
```

Comment:
What if we use something like [`loading.make_indicator`](https://github.com/zulip/zulip/blob/ef49c29ddf41ec660bce9a54f0aa1bb69de5f345/web/src/settings_users.js#L466C5-L466C27) for this?

Reply:
Yes we could use that!